src/main/scala/org/broadinstitute/dsde/rawls/expressions/SlickExpressionParsing.scala

```diff
@@ -113,34 +117,43 @@ trait SlickExpressionParser extends JavaTokenParsers {
   private def entityRootDot: Parser[RootFunc] =
     "this." ^^ { _ => entityRootFunc}
 
+  private val namespacedIdent: Parser[String] = rep1sep(ident, ":") ^^ {
```

Comment:
The `rep1sep` parser will handle strings like one:two:three but this will not match the cases below.  I haven't tested this yet.